docs/dev-tools/plane-compose.md

```
@@ -0,0 +1,539 @@
+### Initialize a project
+
+Create a new project directory with the default structure:
+
+```bash
+plane init my-project
+cd my-project
+```
+
+This creates:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)